components/dashboard/src/repositories/detail/prebuilds/PrebuildSettingsForm.tsx

```diff
@@ -17,15 +21,26 @@ import { LoadingButton } from "@podkit/buttons/LoadingButton";
 import { InputFieldHint } from "../../../components/forms/InputFieldHint";
 import { DEFAULT_WS_CLASS } from "../../../data/workspaces/workspace-classes-query";
 import { Select, SelectItem, SelectTrigger, SelectValue, SelectContent } from "@podkit/select/Select";
+import Alert from "../../../components/Alert";
+import { useUserLoader } from "../../../hooks/use-user-loader";
+import { useUpdateCurrentUserMutation } from "../../../data/current-user/update-mutation";
+import { trackEvent } from "../../../Analytics";
+import dayjs from "dayjs";
 
 const DEFAULT_PREBUILD_COMMIT_INTERVAL = 20;
 
 type Props = {
     configuration: Configuration;
 };
 
+const COACHMARK_KEY = "new_prebuilds_trigger_notification";
```

Comment:
It's not really a coach mark, but I for some reason couldn't imagine `coachmarksDismissals` being used for anything else 🤦

Reply:
Well, hindsight's 20/20